src/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
@hoyosjs, @sywhang, and @josalem, is there a need to produce a netcoreapp2.1 version of the client library? My understanding is that this is not supposed to be an executable entrypoint but just a library to include in other executable or library projects. The reason why I removed this is that it causes a NETSDK1061 error when any other project that references it attempts to do a self-contained publish. It seems that, for example, if a netcoreapp3.1 project references this project, it will try to pull in the netcoreapp2.1 version of the assembly during publish.

If I revert this change, I will have to add additional properties to enable roll forward for the assembly. Not sure you really want to support roll forward for this assembly.

Another option that I can explore is adding yet another target framework for netcoreapp3.1 and see if that resolves the issue. Although, that will bloat the NuGet for this assembly with another assembly that really is equivalent to the netcoreapp2.1 version.